papaya-core/src/main/java/pl/edu/agh/papaya/service/availability/AvailabilityService.java

```diff
@@ -0,0 +1,30 @@
+package pl.edu.agh.papaya.service.availability;
+
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import pl.edu.agh.papaya.model.Availability;
+import pl.edu.agh.papaya.repository.AvailabilityRepository;
+import pl.edu.agh.papaya.service.userinproject.UserInProjectService;
+
+@Service
+@RequiredArgsConstructor
+@SuppressWarnings({"PMD.BeanMembersShouldSerialize"})
+public class AvailabilityService {
+
+    private final AvailabilityRepository availabilityRepository;
+
+    private final UserInProjectService userInProjectService;
+
+    public Optional<Availability> getBySprintIdAndUserId(Long sprintId, Long userId) {
+        return availabilityRepository.findBySprintIdAndUserId(sprintId, userId);
+    }
+
+    public AvailabilityCreationWizard newAvailability() {
+        return new AvailabilityCreationWizard(this, userInProjectService);
+    }
+
+    public Availability save(Availability availability) {
```

Comment:
W tych innych dwóch około-wizardowych przypadkach mamy w serwisie metody typu `createAvailability`

Reply:
I są package-private w sumie

Reply:
A dobra, mnie przerosło czytanie na chwilę - w sumie ma prawo się nazywać `save`, bo tylko to robi

Reply:
O, o, chociaż tutaj mamy tak: https://github.com/papaya-agh/papaya-backend/blob/483e136dbd114aa1c91d8330ca36ca2ed7752a1d/papaya-core/src/main/java/pl/edu/agh/papaya/service/sprint/SprintService.java#L82

Reply:
KONTROWERSYJNA SPRAWA

Reply:
doobra tam, klikam